### PR TITLE
Update releases doc

### PR DESCRIPTION
**Pull request by dmcgowan**
This file does not sync automatically. We might want to consider a way to keep the website updated from the other repos, but for now this brings it up to date.

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

File: content/releases.md [hunk 1]
```diff
@@ -80,13 +80,16 @@ by `<major>.<minor>`. Releases branches will be in one of several states:
 
 - __*Next*__: The next planned release branch.
 - __*Active*__: The release is currently supported and accepting patches.
+- __*Extended*__: The release is only accepting security patches.
```
Comment (mikebrow) [inline content/releases.md]: /s/release/release branch/
and the line above.. and a few below. Alternatively we could explicitly explain the relationship between the term "release" and "release branch".

File: content/releases.md [hunk 2]
```diff
@@ -268,6 +274,13 @@ may be completely refactored or have breaking changes in _minor_ releases.
 
 We will try not break the tool in _patch_ releases.
 
+### Daemon Configuration
+
+The daemon's configuration file, commonly located in `/etc/containerd/config.toml`
+is versioned and backwards compatible.  The `version = 2` field in the config
```
Comment (mikebrow) [inline content/releases.md]: The `version` field in the config file specifies the config's version. If no version number... Use `version = 2` to enable/access ....

File: content/releases.md (hunk 1 above)
Comment (dmcgowan) [reply content/releases.md]: These changes should be made on the original doc, this is only resolving a diff.

**Pull request merged**